tests/integration/test_lambda.py

```diff
@@ -297,13 +298,44 @@ def test_prime_and_destroy_containers():
     status = executor.get_docker_container_status(func_arn)
     assert status == 1
 
+    # assert container network
+    container_network = executor.get_docker_container_network(func_arn)
+    assert container_network == 'default'
+
     executor.cleanup()
     status = executor.get_docker_container_status(func_arn)
     assert status == 0
 
     assert len(executor.get_all_container_names()) == 0
 
 
+def test_docker_command_for_separate_container_lambda_executor():
+
+    # run these tests only for the "separate containers" Lambda executor
+    if not isinstance(lambda_api.LAMBDA_EXECUTOR, lambda_executors.LambdaExecutorSeparateContainers):
+        return
+
+    executor = lambda_api.LAMBDA_EXECUTOR
+    func_name = 'test_docker_command_for_separate_container_lambda_executor'
+
+    func_arn = lambda_api.func_arn(func_name)
+
+    handler = 'handler'
+    lambda_cwd = '/app/lambda'
+    network = 'compose_network'
+
+    config.LAMBDA_DOCKER_NETWORK = network
+
+    cmd = executor.prepare_execution(func_arn, {}, LAMBDA_RUNTIME_NODEJS810, '', handler, lambda_cwd)
+
+    expected = 'docker run -v "%s":/var/task   --network="%s"  --rm "lambci/lambda:%s" "%s"' % (
+        lambda_cwd, network, LAMBDA_RUNTIME_NODEJS810, handler)
+
+    assert cmd == expected, 'cmd=%s expected=%s' % (cmd, expected)
```

Comment:
nit: Instead of doing an exact string comparison here, could we do something like:
```
assert ('--network="%s"' % network) in cmd, 'cmd=%s expected=%s' % (cmd, expected)
```